samples/net/openthread/ot_cli/src/main.c

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+#include <zephyr.h>
+#include <logging/log.h>
+
+LOG_MODULE_REGISTER(cli_sample, CONFIG_OT_COMMAND_LINE_INTERFACE_LOG_LEVEL);
+
+
+void main(void)
+{
+	LOG_INF("OpenThread Command Line Interface is now running.");
+	LOG_INF("Use the 'ot' keyword to invoke OpenThread commands e.g. "
+		"'ot thread start.'");
+	LOG_INF("For the full commands list refer to the OpenThread CLI");
+	LOG_INF("documentation at:");
+	LOG_INF("https://github.com/openthread/openthread/blob/master/src/cli/README.md");
```

Comment:
Why not use only one LOG_INF for this?

Reply:
changed as requested. Added some extra newlines to make it more visible:

uart:~$ *** Booting Zephyr OS build v2.1.99-ncs1-1145-g13dc96f83bab  ***


[00:00:00.467,498] <inf> net_l2_openthread: State changed! Flags: 0x00000010 Current role: 1
[00:00:00.467,529] <inf> net_l2_openthread: State changed! Flags: 0x00000020 Current role: 1
[00:00:00.681,457] <inf> net_config: Initializing network
[00:00:00.806,457] <inf> net_l2_openthread: State changed! Flags: 0x000010e4 Current role: 3
[00:00:00.822,631] <inf> net_l2_openthread: State changed! Flags: 0x00000200 Current role: 3
[00:00:01.581,695] <inf> cli_sample:

OpenThread Command Line Interface is now running.
Use the 'ot' keyword to invoke OpenThread commands e.g. 'ot thread start.'
For the full commands list refer to the OpenThread CLIdocumentation at:
https://github.com/openthread/openthread/blob/master/src/cli/README.md

uart:~$